src/main.js

```diff
@@ -3,83 +3,33 @@
 import Vue from 'vue'
 import App from './App'
 import router from './router'
-import VueMaterial from 'vue-material'
 import Vue2Filters from 'vue2-filters'
-import Toasted from 'vue-toasted';
 import AsyncComputed from 'vue-async-computed'
-import VueTimeline from "@growthbunker/vuetimeline";
+import store from './store';
+import globalMixin from './mixins/global';
+import setupFilters from './utils/filters';
+import 'vue-loaders/dist/vue-loaders.css';
+import VueLoadersBallBeat from 'vue-loaders/dist/loaders/ball-beat';
+import Toast from "vue-toastification";
+import "vue-toastification/dist/index.css";
+import "./styles/overrides.scss";
 
-const CoinGecko = require('coingecko-api');
-const CoinGeckoClient = new CoinGecko();
+Vue.config.productionTip = false;
 
-Vue.use(Vue2Filters)
-Vue.config.productionTip = false
-Vue.use(VueMaterial)
-Vue.use(Toasted)
-Vue.use(AsyncComputed)
-Vue.use(VueTimeline);
+Vue.use(Vue2Filters);
+Vue.use(AsyncComputed);
```

Comment:
I think it's not a good practice to use promises inside computed properties. Would it be difficult to get rid of it by redesigning some components logic?

Reply:
I believe it's not used anymore, removing